src/components/quickfix/QuickFixPanel.tsx

```
@@ -0,0 +1,372 @@
+          <Eye className="h-4 w-4" />
+          {showPreview ? 'Hide Preview' : 'Preview Changes'}
+        </button>
+
+        {showPreview && preview && (
+          <div className="border border-gray-200 rounded-lg overflow-hidden">
+            <div className="bg-gray-50 px-3 py-2 border-b border-gray-200">
+              <span className="text-sm font-medium text-gray-700">Code Changes</span>
+            </div>
+            <div 
+              className="p-3 bg-gray-900 text-sm font-mono overflow-x-auto"
+              dangerouslySetInnerHTML={{ __html: preview.diff }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)